content/200-concepts/050-overview/500-under-the-hood.mdx

```diff
@@ -75,22 +77,31 @@ Run the following command to set the environment variable globally (in this exam
 
    <tab>
 
-   ```terminal
-   export PRISMA_QUERY_ENGINE_BINARY=/custom/my-query-engine-unix
-   ```
+```terminal
+export PRISMA_QUERY_ENGINE_BINARY=/custom/my-query-engine-unix
+```
 
    </tab>
 
    <tab>
 
-   ```terminal
-   set PRISMA_QUERY_ENGINE_BINARY=c:\custom\my-query-engine-windows.exe
-   ```
+```terminal
+set PRISMA_QUERY_ENGINE_BINARY=c:\custom\my-query-engine-windows.exe
+```
 
    </tab>
 
 </TabbedContent>
 
+#### Navigating a corperate firewall with <inlinecode>PRISMA_BINARIES_MIRROR</inlinecode>
+
+You might find that your company operates behind a corporate firewall and you are unable to access the Prisma binaries via the default url `https://binaries.prisma.sh`.
+In such situations the `PRISMA_BINARIES_MIRROR` environment variable can be used to bypass the corporate firewall by proxying the request to the Prisma domain and download the binaries.
+
+```terminal
+PRISMA_BINARIES_MIRROR=https://my-whitelisted-url
```

Comment:
We prefer using alternatives to blacklist / whitelist. (racially insensitive terms).
Maybe change to allow / allowed

Reply:
Check everywhere in PR to replace all occurences